ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -199,18 +199,18 @@ const triggerSet: OopsyTriggerSet<Data> = {
       netRegex: NetRegexes.ability({ id: raiseAbiliyIds }),
```

Comment:
What about:  `netRegex: NetRegexes.ability({ id: raiseAbiliyIds, targetId: 'E0000000' }),` and then you don't need an extra check.

(Also, I know this was there before, but can you rename `raiseAbiliyIds` to `raiseAbilityIds`?)